draft-xie-bidirectional-messaging-latest.md

```diff
@@ -113,26 +113,26 @@ apply to this document.
 
 ## RStream and XStream
 
-A routing stream (RStream) is a regular HTTP/2 stream in nature. It is opened by
-a HEADERS frame, and **MAY** be continued by CONTINUATION and DATA frames.
-RStreams are initiated by clients to servers, and can be independently routed by
-intermediaries on the network path. The main purpose for RStream is to
+A routing stream (RStream) is a regular HTTP/2 stream. It is opened by a HEADERS
+frame, and **MAY** be continued by CONTINUATION and DATA frames. RStreams are
+initiated by clients to servers, and can be independently routed by
+intermediaries on the network path. The main purpose for an RStream is to
 facilitate XStreams' intermediary traversal.
 
 A new HTTP/2 stream called eXtended stream (XStream) is introduced for
 exchanging user data bidirectionally. An XStream is opened by an XHEADERS frame,
 and **MAY** be continued by CONTINUATION and DATA frames. XStreams can be
-initiated by either clients or servers. Another difference from a regular
-stream, an XStream **MUST** be associated with an open RStream. In this way,
-XStreams can be routed according to their RStreams by intermediaries and
-servers. XStream **MUST NOT** be associated with any other XStream, or any
-closed RStream. Otherwise, it cannot be routed properly.
+initiated by either clients or servers and every XStream **MUST** be associated
```

Comment:
Purely style: Can we skip bold on all the 2119 keywords and just go with all caps?

Reply:
I thought **CAPS** is to help xml2rfc tools, for generating proper rfc format text files.   Otherwise, we can just remove the **.  no objection.

Reply:
I think it should work okay with CAPS and no `**`, just wanted to make sure that wasn't a style preference. I'll go through and swap that out once we get through with the more substantive changes.